package.json

```
@@ -39,7 +39,7 @@
   },
   "peerDependencies": {
     "d3": "^5.5.0",
-    "react": "^16.4.1"
+    "react": "^17.0.2"
```

I'm ok on allowing react 16 or greater, but we shouldn't block clients to use react 16.

```suggestion
    "react": ">=16.4.1"
```

---

you could create a new version for this